cloud_automation/java/dataflow/src/main/java/org/datacommons/dataflow/CsvImport.java

```diff
@@ -0,0 +1,121 @@
+package org.datacommons.dataflow;
+
+import com.google.bigtable.v2.Mutation;
+import com.google.bigtable.v2.Mutation.SetCell;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.ByteString;
+import org.apache.beam.runners.dataflow.options.DataflowPipelineOptions;
+import org.apache.beam.sdk.io.TextIO;
+import org.apache.beam.sdk.io.gcp.bigtable.BigtableIO;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.options.ValueProvider;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.PipelineResult;
+import org.apache.beam.sdk.transforms.MapElements;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.beam.sdk.values.KV;
+import org.apache.beam.sdk.values.TypeDescriptor;
+import org.apache.hadoop.hbase.util.Bytes;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*
+
+public class CsvImport {
+
+  private static final byte[] FAMILY = Bytes.toBytes("csv");
+  private static final Logger LOG = LoggerFactory.getLogger(CsvImport.class);
+
+  static class CsvToBigtableFn
+      extends DoFn<String, KV<ByteString, Iterable<Mutation>>> {
+    @ProcessElement
+    public void processElement(ProcessContext c) throws Exception {
+      try {
+        // TODO: Pass header as congfiguration.
+        String[] headers = new String[] {"key", "value"};
+        String[] values = c.element().split(",");
+        Preconditions.checkArgument(headers.length == values.length);
+
+        byte[] rowkey = Bytes.toBytes(values[0]);
+        byte[][] headerBytes = new byte[headers.length][];
+        ImmutableList.Builder<Mutation> mutations = ImmutableList.builder();
+        for (int i = 0; i < headers.length; i++) {
+          Mutation.SetCell setCell = null;
+          headerBytes[i] = Bytes.toBytes(headers[i]);
+          setCell =  Mutation.SetCell.newBuilder()
+                          .setFamilyName(FAMILY)
+                          .setColumnQualifier(ByteString.copyFrom( headerBytes[i]))
+                          .setValue(ByteString.copyFrom(Bytes.toBytes(values[i])))
+                          .build();
+           mutations.add(Mutation.newBuilder().setSetCell(setCell).build());
+        }
```

Comment:
Do we need to set "key" as a column if it is anyway going to be in row-key?

(sorry never really read this code before)

Reply:
I was not sure about the downstream impact of this, so I tried to keep the BT schema same. @Spaceenter do you know if this would work?

Reply:
In any case, this is for a potential follow-up change.

Feel free to submit this.